examples/lazy-injection/counter-todo/src/app/hooks.ts

```diff
@@ -0,0 +1,7 @@
+import type { TypedUseSelectorHook } from "react-redux"
+import { useDispatch, useSelector } from "react-redux"
+import type { AppDispatch, RootState } from "./store"
+
+// Use throughout your app instead of plain `useDispatch` and `useSelector`
+export const useAppDispatch: () => AppDispatch = useDispatch
+export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector
```

Comment:
```suggestion
import { useDispatch, useSelector } from "react-redux"
import type { AppDispatch, RootState } from "./store"

// Use throughout your app instead of plain `useDispatch` and `useSelector`
export const useAppDispatch = useDispatch.withTypes<AppDispatch>()
export const useAppSelector = useSelector.withTypes<RootState>()
```